lib/pry/commands/hist.rb

```diff
@@ -170,8 +170,9 @@ def find_history
           else
             Pry.history.to_a.last(Pry.history.session_line_count)
           end
-      # The last value in history will be the 'hist' command itself.
-      Pry::Code(h[0..-2])
+
+      # The last value in history will be the 'hist' command itself
```

Comment:
ok?  #The last value in history will be the 'hist' command itself unless in histignore.